Greenline/ConfigExtractor.cs

```diff
@@ -55,8 +55,22 @@ public static void TryParseConfig(TypeDefinition type, out string? config)
 
     private static string Decrypt(string input, string key)
     {
-        string first = FromBase64(input);
-        return FromBase64(Xor(first, key));
+        bool Base64String = IsBase64String(input);
+        if (Base64String)
+        {
+            string first = FromBase64(input);
+            return FromBase64(Xor(first, key));
+        }
+        else
```

Comment:
We can cut the else block here and do just the return since the return in the if will abort further execution anyways:
 
 ```csharp
        if (Base64String)
        {
            string first = FromBase64(input);
            return FromBase64(Xor(first, key));
        }
        
        return input;
 ```

This would be nice to comply with the code style

Reply:
Agree! that is True. Will Follow it.